src/types/types.d.ts

```diff
@@ -20,11 +20,30 @@ export interface ButtonProps {
 }
 
 export interface ISetKeyPair {
-  setSecret: (secret: string | null) => void;
+  setSecretKey: (secret: string | null) => void;
   setPublicKey: (publicKey: string | null) => void;
 }
 
 export interface Ikeypair {
```

Comment:
`IKeypair`

Reply:
Solved in commit 33928f3eda3fbbb44395cf9225926437064bc666.